src/_posts/aframe-v0.3.0.md

```diff
@@ -0,0 +1,130 @@
+---
+title: A-Frame v0.3.0 - Walk in a Web Page
+author: twitter|andgokevin|Kevin Ngo
+date: 2016-07-25 10:00:00
+layout: blog
+image:
+  src: aframe-v0.3.0.png
+---
+
+**v0.3.0** has dropped! *What is virtual reality without our legs nor our
```

Comment:
I'd start with "A-Frame v0.3.0 has dropped" ; starting a sentence with "v" and a few numbers and punctution looks odd for some reason.